src/modules/extendedCallList/assets/fixedEventInfo.ts

```diff
@@ -0,0 +1,34 @@
+export default (LSSM: Vue) => {
+    const eventInfo = document.querySelector<HTMLDivElement>('#eventInfo');
+    if (!eventInfo) return;
+
+    const whitespace = document.createElement('div');
+    whitespace.id = LSSM.$store.getters.nodeAttribute(
+        'ecl-fixed_event_info-whitespace'
+    );
+    whitespace.style.setProperty('height', getComputedStyle(eventInfo).height);
+    eventInfo.after(whitespace);
+
+    LSSM.$store
+        .dispatch('addStyle', {
+            selectorText: `#eventInfo`,
+            style: {
+                'position': 'absolute',
+                'width': 'calc(100% - 2 * (15px + 1px) - 18px)',
```

Comment:
Does this work with very small/big displays?

Reply:
yes. It is exactly the width of the missions list, including some padding. The event-Info wil lhave the exact same width as missions